app/assets/src/components/common/DetailsSidebar/SampleDetailsMode/PipelineTab.jsx

```diff
@@ -105,24 +105,28 @@ class PipelineTab extends React.Component {
         >
           <div className={cs.downloadSectionContent}>
             {pipelineRun &&
-              getDownloadLinks(sampleId, pipelineRun).map(option => (
-                <a
-                  key={option.label}
-                  className={cs.downloadLink}
-                  href={option.path}
-                  target={option.newPage ? "_blank" : "_self"}
-                  onClick={() =>
-                    logAnalyticsEvent("PipelineTab_download-link_clicked", {
-                      newPage: option.newPage,
-                      label: option.label,
-                      href: option.path,
-                      sampleId: this.props.sampleId,
-                    })
-                  }
-                >
-                  {option.label}
-                </a>
-              ))}
+              getDownloadLinks(sampleId, pipelineRun).map(
+                option =>
+                  (option.label != "See Pipeline Visualization" ||
```

Comment:
Lets pass `allowedFeatures` instead to the `getDownloadLinks` and have that function only add the download link if the feature is enabled?